staging/src/k8s.io/csi-translation-lib/translate.go

```diff
@@ -147,7 +147,7 @@ func (CSITranslator) GetInTreePluginNameFromSpec(pv *v1.PersistentVolume, vol *v
 				return curPlugin.GetInTreePluginName(), nil
 			}
 		}
-		return "", fmt.Errorf("could not find in-tree plugin name from persistent volume %v", pv)
+		return "", fmt.Errorf("could not find in-tree plugin name from persistent volume %s:%s", pv.Namespace, pv.Name)
```

Comment:
I'm a little confused about how this change (and the change in `volume_expand.go` are related). Can you share more about the connection?

Reply:
Change to volume_expand.go would be taken out.

The above change facilitates debugging in IDE - if possible, we should keep it.